joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50.0

arm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - joint_1
    - joint_2
    - joint_3
    - joint_4
    - joint_5
    - joint_6
  gains:
    joint_1: {p: 200, d: 10, i: 0}
    joint_2: {p: 200, d: 10, i: 0}
    joint_3: {p: 150, d: 8,  i: 0}
    joint_4: {p: 80,  d: 3,  i: 0}
    joint_5: {p: 50,  d: 2,  i: 0}
    joint_6: {p: 30,  d: 1,  i: 0}
    # finger_joint: {p: 0, d: 0, i: 0}

r2_arm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - r2_joint_1
    - r2_joint_2
    - r2_joint_3
    - r2_joint_4
    - r2_joint_5
    - r2_joint_6
  gains:
    r2_joint_1: {p: 200, d: 10, i: 0}
    r2_joint_2: {p: 200, d: 10, i: 0}
    r2_joint_3: {p: 150, d: 8,  i: 0}
    r2_joint_4: {p: 80,  d: 3,  i: 0}
    r2_joint_5: {p: 50,  d: 2,  i: 0}
    r2_joint_6: {p: 30,  d: 1,  i: 0}


# dual_arm_controller:
#   type: position_controllers/JointTrajectoryController
#   joints:
#    joints:
#     - joint_1
#     - joint_2
#     - joint_3
#     - joint_4
#     - joint_5
#     - joint_6
#     - finger_joint
#     - r2_joint_1
#     - r2_joint_2
#     - r2_joint_3
#     - r2_joint_4
#     - r2_joint_5
#     - r2_joint_6
#     - r2_finger_joint
#   gains:
#     joint_1: {p: 200, d: 10, i: 0}
#     joint_2: {p: 200, d: 10, i: 0}
#     joint_3: {p: 150, d: 8,  i: 0}
#     joint_4: {p: 80,  d: 3,  i: 0}
#     joint_5: {p: 50,  d: 2,  i: 0}
#     joint_6: {p: 30,  d: 1,  i: 0}
#     r2_joint_1: {p: 200, d: 10, i: 0}
#     r2_joint_2: {p: 200, d: 10, i: 0}
#     r2_joint_3: {p: 150, d: 8,  i: 0}
#     r2_joint_4: {p: 80,  d: 3,  i: 0}
#     r2_joint_5: {p: 50,  d: 2,  i: 0}
#     r2_joint_6: {p: 30,  d: 1,  i: 0}
#     r2_finger_joint: {p: 5, d: 0.5, i: 0}
#     finger_joint: {p: 5, d: 0.5, i: 0}



gripper_controller:
  type: position_controllers/JointPositionController
  joint: finger_joint
  pid: {p: 5, d: 0.5, i: 0}

r2_gripper_controller:
  type: position_controllers/JointPositionController
  joint: r2_finger_joint
  pid: {p: 5, d: 0.5, i: 0}
